Author: Michael Webster <miketwebster@gmail.com>
Date: Wed, 9 Aug 2017 19:54:26 -0400
Description: nemo.desktop: Remove DBusActivatable line - we don't actually
 support it yet, it was an unnecessary change in ad5fcfc747466715d.  This ends
 up just causing issues with getting nemo to launch in more up-to-date
 distributions.
Origin: upstream, https://github.com/linuxmint/nemo/commit/66d7aae2ee00a29276cbbf97491e7455a5cba137
Last-Update: 2017-09-05

---

 data/nemo.desktop.in.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/nemo.desktop.in.in b/data/nemo.desktop.in.in
index e0324d781..13c4e8151 100644
--- a/data/nemo.desktop.in.in
+++ b/data/nemo.desktop.in.in
@@ -110,7 +110,6 @@ Icon=folder
 Terminal=false
 Type=Application
 StartupNotify=false
-DBusActivatable=true
 Categories=GNOME;GTK;Utility;Core;
 MimeType=inode/directory;application/x-gnome-saved-search;
 Actions=open-home;open-computer;open-trash;
